src/main/java/com/lob/protocol/response/AbstractPagedResponseList.java

```diff
@@ -8,7 +8,7 @@
 
 import static com.lob.Util.defensiveCopy;
 
-public abstract class AbstractPagedResponseList<T> implements Iterable<T> {
+public abstract class AbstractPagedResponseList<T> extends AbstractResponse implements Iterable<T> {
```

Comment:
No `AbstractPagedResponseList` is extending `AbstractResponse` AND implementing the properties of `Iterable` which will give you access to functions and properties like `.hasNext()`, `.next()`, `.remove()` or even `for (Object o: pagedResponse) { ... }`